lib/types.ts

```diff
@@ -32,29 +32,84 @@ export interface PluginMetadata {
   imageLayers: string[];
 }
 
-export interface Package {
+export interface PluginResponseStatic extends PluginResponse {
+  hashes: string[];
+}
+
+// NEW STUFF
+
+export interface PluginResponse {
+  plugin: PluginMetadata;
+  scannedProjects: ScannedProjectCustom[];
+}
+
+export interface ScannedProjectCustom {
+  packageManager: string; // actually SupportedPackageManagers; in the CLI
+  depTree: DepTree;
+  targetFile?: string;
+  meta?: any;
+}
+
+export interface DepTreeDep {
+  name: string;
+  version: string;
   dependencies: {
-    [key: string]: any; // TODO
+    [depName: string]: DepTreeDep;
   };
-  docker: {
-    [key: string]: any; // TODO
+  labels?: {
+    [key: string]: string;
   };
-  name: string;
+}
+
+export interface DepTree extends DepTreeDep {
```

Comment:
Are these two interfaces present in some shared library we could use? They seem oddly familiar!

Reply:
they are written and rewritten everywhere 😭and the current advice is not to use a shared library.
you can find them in http://github.com/snyk/snyk-cli-interface, https://github.com/snyk/snyk, and some of our backend services I believe in different forms.

Reply:
![image](https://user-images.githubusercontent.com/16254901/80116504-f8374780-857d-11ea-9e64-fb72672949c9.png)

They exists on the cli-interface module, but I think @Shesekino doesnt want to have cli-interface import here